sample/res/layout/activity_place_sample.xml

```diff
@@ -5,21 +5,48 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-  <!-- ListView places -->
-  <ListView
-      android:id="@+id/lv_places"
-      android:layout_width="fill_parent"
-      android:layout_height="fill_parent"/>
-
-  <!-- DraggablePanel -->
-
-  <com.github.pedrovgs.DraggablePanel
-      xmlns:draggable_panel="http://schemas.android.com/apk/res-auto"
-      android:id="@+id/draggable_panel"
-      android:layout_width="fill_parent"
-      android:layout_height="fill_parent"
-      draggable_panel:enable_click_to_maximize_panel="true"
-      draggable_panel:enable_click_to_minimize_panel="true"/>
+  <android.support.v4.widget.DrawerLayout
```

Comment:
If you push a new commit in the pull request branch GitHub should update this pull request automatically.